src/applications/facility-locator/components/search-results/LocationAddress.jsx

```diff
@@ -4,6 +4,21 @@ import { buildAddressArray } from '../../utils/facilityAddress';
 
 const LocationAddress = ({ location }) => {
   const addressArray = buildAddressArray(location);
+  if (location.resultItem) {
+    if (addressArray.length === 0) {
+      return (
+        <dd>
+          <strong>Address: </strong>
+          Contact for Information
+        </dd>
+      );
+    }
+    return (
+      <dd>
+        {[].concat(...addressArray.map(e => [<br key={e} />, e])).slice(1)}
+      </dd>
```

Comment:
I think you could omit the `<br/>` element here through a couple VADS utilities. It also seems like iterating through the address should render a list element but I'm not sure.

```jsx
return (
    <dd>
        <ul className="usa-unstyled-list">
          {addressArray
               .slice(1) // Why are we slicing?
                .map(address => (
                   <li key={address} className="vads-u-border-top--1px vads-u-border-color--gray-lighter vads-u-padding-top--3"> 
                     {address} 
                   </li>
                )}
        </ul>
    </dd>
);
```

`.map` and `.slice` don't mutate the original array so you don't have to worry about the `[].concat`.